app/src/main/java/org/cru/godtools/ui/tooldetails/ToolDetailsFragment.kt

```diff
@@ -57,6 +57,7 @@ class ToolDetailsFragment() : BasePlatformFragment<ToolDetailsFragmentBinding>(R
         super.onBindingCreated(binding, savedInstanceState)
         binding.fragment = this
         binding.tool = dataModel.tool
+        binding.hasTraining = dataModel.hasTraining
```

Comment:
@frett I just curious why we are adding things one by one instead of just adding the dataModel.

Reply:
it actually makes testing of the data binding much easier.

If I wanted to test the behavior of `hasTraining` I could provide a mock instance of that without needing to create a full data model object.

Reply:
take a look at https://github.com/CruGlobal/godtools-android/blob/develop/app/src/test/java/org/cru/godtools/databinding/ToolsListItemToolBindingTest.kt to see how setting individual LiveData objects on the binding allows us to more easily test behavior without having to mock the entire data model

Reply:
also, the data binding layouts are a bit more portable if it depends on 4 LiveData objects vs a data model that is more tightly coupled to a single fragment or activity.